File: Anul-2/Semestrul I/MAP/Seminar/Seminar8/src/main/java/org/example/seminar8/Student.java
package org.example.seminar8;

public class Student extends Entity<Long>{
private String name;
private int group;

    @Override
    public String toString() {
        return "Student{" +
                "name='" + name + '\'' +
                ", group=" + group +
                '}';
    }

    public Student(String name, int group) {
        this.name = name;
        this.group = group;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getGroup() {
        return group;
    }

    public void setGroup(int group) {
        this.group = group;
    }
}

File: Anul-2/Semestrul I/MAP/Laborator/Laborator-10/Laborator-10/domain/Student.cs
namespace Laborator_10.domain;

public class Student:Entity<int>
{
    private string _name;
    private string _schoolName;

    public Student(string name, string schoolName)
    {
        this._name = name;
        this._schoolName = schoolName;
    }

    public string GetName()
    {
        return _name;
    }

    public string GetSchoolName()
    {
        return _schoolName;
    }

    public override string ToString()
    {
        return $"Name: {_name}, SchoolName: {_schoolName}";
    }
}